### ISSUE TITLE
Categories should be manually sortable

### ISSUE THREAD

**Issue by mindeffects** [modx-ccc-2015/whishlist#29]
Problem:
When using categories to structure elements, it nearly never happens, that the order "alphabetically" comes in handy. Most of the time you have to use numbers or so to bring the categories into the desired order.

Solution:
Categories should be manually sortable.

**Comment by JensWolff** [comment]
+1
That would be very helpful, if they could be reordered differently for each template by drag'n'drop

**Comment by DESIGNfromWITHIN** [comment]
Yes, would be good a field similar to menu number  would do it right?

**Comment by theboxer** [comment]
@mindeffects this is implemented for 2.4, categories have rank field. The rank field is used when showing categories in TV tab/panel in Resources.

**Comment by mindeffects** [comment]
Cool, @TheBoxer! Incl. "drag and drop" re-arranging or "typing numbers like 1999"? ;-)

**Comment by theboxer** [comment]
Typing numbers, the rank is not used in the element's tree

**Comment by mindeffects** [comment]
What a pity. I was about to closing this thread. ;-)

I would ALLWAYS manage hierarchies visually, because this way it is WAY faster and easier to arrange groups and subgroups: Drag and drop stuff just like resources/containers.
